docs/en/spark/configuration/source-plugins/Ignite.md

```diff
@@ -0,0 +1,41 @@
+# Source plugin: Ignite
+
+### Description
+
+Read data from Ignite.
+
+### Options
+
+| name           | type   | required | default value |
+| -------------- | ------ | -------- | ------------- |
+| [table](#table-string)       | string | yes      | -             |
+| [config.file.path](#config.file.path-string)        | string | yes      | -         |
+
```

Comment:
If I remember correctly `result_table_name` seems to be required, please check, thanks